setup.py

```diff
@@ -21,9 +21,9 @@
         "black>=22,<23",
     ],
     "doc": [
-        "Sphinx>=1.6.5,<5",
-        "jinja2>=3.0.0,<3.1.0",  # jinja2<3.0 or >=3.1.0 cause doc build failures.
-        "sphinx_rtd_theme>=0.1.9,<1",
+        "Sphinx>=4.2.0",
+        "jinja2>=3.0.0",  # jinja2<3.0 or >=3.1.0 cause doc build failures.
```

Comment:
we may be able to remove this line in general?

Reply:
If we don't use Jinja at all, then for sure. I'll try it. This is where I went down the RTD webhook refresh rabbithole.